designs/octoplushy/src/head.mjs

```diff
@@ -613,15 +613,10 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
-    toAbs: (val, { options }, mergedOptions) =>
-      ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
-      (options.type === undefined
-        ? 1
-        : options.type == 'octopus'
-        ? 1.7
-        : options.type == 'squid'
-        ? 2
-        : 1),
+    // eslint-disable-next-line no-empty-pattern
+    toAbs: (val, {}, mergedOptions) =>
```

Comment:
No need to destructure the second parameter. You can use `_` like on like 628 so that we're consistent.